RELEASE/scripts/autoscend/combat/auto_combat_util.ash

```diff
@@ -553,6 +553,10 @@ string banisherCombatString(monster enemy, location loc, boolean inCombat)
 		return "skill " + $skill[Howl of the Alpha];
 	}
 
+	if(item_amount($item[Handful of split pea soup]) > 0 && (!(used contains "Handful of split pea soup")) && auto_is_valid($item[Handful of split pea soup]) && useFree)
```

Comment:
if not in combat, check createable_amount + item_amount > 0. This way we use the adjust for banisher code you added to create it if we don't have any on hand